README.md

```diff
@@ -118,7 +118,7 @@ If you do not have `make`, you can do the following:
 ```sh
 $ ocaml bootstrap.ml
 $ ./dune.exe build -p dune --profile dune-bootstrap
-$ ./dune.exe install dune --build-dir _boot
+$ ./dune.exe install dune --build-dir _build
```

Comment:
We can now remove the `--build-dir` option entirely

Reply:
I've removed it. Please approve the PR, if everything else is fine.